### PR TITLE
Added Angular Roadmap

### PR DESCRIPTION
**Pull request by AdityaBansal2310**
I have added Angular Roadmap. Look after that and let me know if there are some improvements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
 ## Index
 
 - [Android Development](#android-development)
+- [Angular](#angular)
```
Comment (Ashish-khanagwal) [inline README.md:48]: Angular section has already been added to the index, delete this as well.
make sure to sync your local repo before contributing to ignore merge conflicts.

File: README.md [hunk 2]
```diff
@@ -112,6 +113,189 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
...
+18. Mobile Application Development
+
+    - [NativeScript](https://www.nativescript.org/)
+    - [Ionic](https://ionicframework.com/)
+
+19. Desktop Application Development
+
+    - [Angular Universal](https://universal.angular.io/)
+    - [Electron](https://electronjs.org/)
+
```
Comment (Ashish-khanagwal) [inline README.md:296]: don't use numbers, 
use `##` for heading and `-` to generate list

**Review by Ashish-khanagwal — APPROVED** [review]
.

**Review by Ashish-khanagwal — CHANGES_REQUESTED** [review]
Hey, Some of the resources are already added to this repo, Please carefully look at the repository before adding resources.
You don't have to add HTML, CSS, and JavaScript submodules as they are also added in the repo.

**Comment by Ashish-khanagwal** [comment]
@AdityaBansal2310 Please carefully look at the markdown used while adding resources and follow the instructions.

File: README.md [hunk 3]
```diff
@@ -112,6 +113,189 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
     
 ---  
 
+## Angular 
+1. IDEs
+
```
Comment (Ashish-khanagwal) [inline README.md:118]: Don't use numbers list, and use proper markdown for headings

For headings - `##`
For list - `-`

**Review by Ashish-khanagwal — CHANGES_REQUESTED** [review]

**Comment by AdityaBansal2310** [comment]
I am working on it and will make another PR by tomorrow keeping in mind all the changes you have asked to do.

**Comment by Ashish-khanagwal** [comment]
So, can I close this PR, as you'll be making another PR

File: README.md [hunk 4]
```diff
@@ -118,64 +118,21 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
...
-    - [jsfiddle](https://jsfiddle.net/)
-    - [plnkr](https://plnkr.co/)
-    - [jsbin](https://jsbin.com/?html,output)
-    - [codepen](https://codepen.io/pen/)
     - [stackblitz](https://stackblitz.com/)
 
-5. Angular
+- Angular
 
     - [official website](https://angular.io/)
```
Comment (Ashish-khanagwal) [inline README.md:129]: please remove these one line spaces in between the heading and its related resources, from the rest of your added resources

**Review by Ashish-khanagwal — CHANGES_REQUESTED** [review]
in line 258, add a line for separation
```
---
```

**Comment by Ashish-khanagwal** [comment]
@AdityaBansal2310 you're PR not following the contribution guidelines. So, closing your PR.
If you want to contribute to this repo, please make sure to read the contribution guidelines carefully.

**Pull request closed without merging**